ci/do_ci.sh

```diff
@@ -419,22 +418,6 @@ elif [[ "$CI_TARGET" == "check_format" ]]; then
   "${ENVOY_SRCDIR}"/tools/code_format/check_format.py check
   BAZEL_BUILD_OPTIONS="${BAZEL_BUILD_OPTIONS[*]}" "${ENVOY_SRCDIR}"/tools/proto_format/proto_format.sh check --test
   exit 0
-elif [[ "$CI_TARGET" == "check_spelling" ]]; then
```

Comment:
hm, do you think we should keep these around for folks who are used to running things this way, or only want to do spelling not all pre checks?

We can also remove it now and add it back if anyone complains but I have these aliased so I'd mildly prefer to keep but I can totally update my aliases too :-)

Reply:
i think we need to remove these - i think the `do_ci.sh` switchboard is a bit of an anti pattern - the related readme has been updated